src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -0,0 +1,780 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Delete the aspnetcoreapplication.cxx file.

Reply:
Going to wait to review this file until the other is deleted. Hard to see the differences 😄